src/views/pages/GenerateKeypair/index.js

```diff
@@ -1,25 +1,21 @@
-import React from 'react';
-import CLI from '../../../services/CLI';
+import React, {useState} from 'react';
 import {CLI_URI} from '../../../config';
 import {Client as WsClient} from 'rpc-websockets';
 import PageTitle from '../../components/PageTitle';
 import Text from '../../components/Text';
 import Button from '../../components/Button';
 import IdentitySubNav from '../../components/IdentitySubNav';
+import useFennelRPC from '../../../utils/useFennelRPC';
```

Comment:
The work I did in whiteflag-demo will break this, but its a quick fix -- if you are able, we can take a quick 15 min to knock this out -- I suggest meeting because the breaking change goes beyond changing the location of the hook.